CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # ChangeLog
+## Release RHEL87_23WW21.5_627.7_23.4.15_PSB_230307.15
+* i915: fix for reported list corruption on flex and ai issue seen on max
+* i915: fix for LMEM->SMEM migration
```

Comment:
3 more changes are missing as per  https://github.com/intel-innersource/drivers.gpu.linux-releases/blob/releases/2023-05-01-ci-devel-97/template/CHANGES.md  from in between release 622